packages/v4/src/pages/home.js

```diff
@@ -0,0 +1,290 @@
+                  <Text
+                    aria-label="view layouts"
+                    className="pf-u-pl-0 pf4-c-link__text ws-mdx-a"
+                  >
+                    <Link to="/documentation/react/layouts/bullseye">
+                      View layouts <ArrowRightIcon />
+                    </Link>
+                  </Text>
+                </div>
+                <div className="pf-u-pb-2xl">
+                  <Text component={TextVariants.h2}>Build things your way</Text>
+                  <Text component={TextVariants.p} className="ws-mdx-p">
+                    Components are like building blocks. Designed to be flexible and modular, you can mix and match components to create a solution for almost any UI problem.
+                  </Text>
+                  <Text
+                    aria-label="view components"
+                    className="pf-u-pl-0 pf4-c-link__text ws-mdx-a"
+                  >
+                    <Link to="/documentation/react/components/aboutmodal">
+                      View components <ArrowRightIcon />
+                    </Link>
+                  </Text>
+                </div>
+                <div className="pf-u-pb-2xl">
+                  <Text component={TextVariants.h2}>See it in action</Text>
+                  <Text component={TextVariants.p} className="ws-mdx-p">
+                    Components are better together. Demos show how multiple components can be used in a single design. They also provide  some useful starter code, so the days of creating complex layouts from scratch are over.
+                  </Text>
+                  <Text
+                    aria-label="view demos"
+                    className="pf-u-pl-0 pf4-c-link__text ws-mdx-a"
+                  >
+                    <Link to="/documentation/react/demos/bulkselecttable">
+                      View demos <ArrowRightIcon />
+                    </Link>
+                  </Text>
+                </div>
+              </TextContent>
+            </GridItem>
+            <GridItem sm={12} md={6}>
+              <div type="image/svg+xml" className="pf4-c-image__laptop" aria-label="laptop image" />
+              <div type="image/svg+xml" className="pf4-c-image__phone" aria-label="phone image" />
+              <div type="image/svg+xml" className="pf4-c-image__screen" aria-label="screen image" />
+              <div type="image/svg+xml" className="pf4-c-image__desktop" aria-label="desktop image" />
+            </GridItem>
+          </Grid>
+        </GridItem>
+      </Grid>
+    </PageSection>
+    <PageSection variant={PageSectionVariants.darker} className="pf4-c-background-image-principles ws-homepage-main-section">
+      <Grid gutter="md" className="pf-u-py-3xl">
+        <GridItem sm={12} md={6} mdOffset={3} className="pf-u-text-align-center">
+          <TextContent>
+            <Text component={TextVariants.h1} className="pf-u-mb-4xl pf-m-white pf-u-text-align-center" id="patternfly-principles">
```

Comment:
I only changed the outermost `<SideNavLayout>` of this component and renamed it from `index.js` to `home.js`. The point of this PR is more to get Webpack working than to add this homepage, but I'll address your comments.

This page will likely get redesigned since the side nav is taking up width.

Reply:
I Made this an h2.